### ISSUE TITLE
Controls: Players should be able to choose custom mapping without side effects 

### ISSUE THREAD

**Issue by salvob41** [notapixelstudio/superstarfighter#946]
After #924 . Players could choose a custom mapping. Maybe esplicitely in the character selection?
